app/uk/gov/hmrc/eoricommoncomponent/frontend/CdsErrorHandler.scala

```diff
@@ -43,10 +43,13 @@ class CdsErrorHandler @Inject() (
 
   private val logger = Logger(this.getClass)
 
+  // $COVERAGE-OFF$This method is unused but need to be defined, because it's not defined in FrontendErrorHandler
```

Comment:
Can't you just call this method from the spec and check the exception is thrown?

Reply:
That's a good point. I did it in separate commit